application.yml

```diff
@@ -18,21 +18,8 @@ dependencies:
       service-environment:
         COLLECTION_NAME: 'test-collection'
 
-
-environments:
-  local:
-  production:
-    domain: spacetweet.originate.com
-    providers:
-      aws:
-        region: us-west-2
-        public-cluster-size: t2.medium
-        private-cluster-size: t2.nano
-        remote-state-store: terraform.state.prod
-  staging:
-    url: dev.megabyt.es
-    provider: aws
-    region: us-west-2
+production:
+  url: spacetweet.originate.com
```

Comment:
Is this needed? Its also listed in the web-server.

Reply:
Yeah, there's an application domain name, and then one for each of the public facing services. Right now they happen to be the same.